core/standalone/README.md

```diff
@@ -204,7 +219,35 @@ Api Gateway mode can be enabled via `--api-gw` flag. In this mode upon launch a
 would be launched on port `3234` (can be changed with `--api-gw-port`). In this mode you can make use of the
 [api gateway][4] support.
 
+#### Using Kafka
+
+Standalone OpenWhisk supports launching an [embedded kafka][5]. This mode is mostly useful for developers working on OpenWhisk
+implementation itself.
+
+```
+java -jar openwhisk-standalone.jar --kafka
+```
+
+It also supports launching a Kafka UI based on [Kafdrop 3][6] which enables seeing the topics created and structure of messages
```

Comment:
`[Kafdrop 3][6]` -> is this supposed to link to two different places, "3" and "6"?

Reply:
No thats the actual name [Kafdrop 3](https://github.com/obsidiandynamics/kafdrop)